.github/workflows/docx-diff.yml

```
@@ -0,0 +1,148 @@
+
+jobs:
+  docx-diff:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write  # Needed to post PR comments
+      contents: read
+
+    steps:
+      - name: Get PR details
+        id: pr
+        run: |
```

actionlint (1.7.10)

[error] "github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)